.github/workflows/crowdin-upload.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           upload_translations: true
           download_translations: false
           project_id: '492549'
-          token: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+          # token: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+          token: '3483c7f56ed2018ec04cf6194de23e1d3552f34bff605bd092728781d0e88ec27d4af1a9ed10e433'
```

Comment:
This isn't our real one, just wanted to make sure I was updating the right thing in Crowdin!